qubesadmin/tools/qvm_create.py

```diff
@@ -131,23 +133,24 @@ def main(args=None, app=None):
             args.properties['label'],
             ', '.join(args.app.labels)))
 
+    cls = args.cls.lower().replace("vm", "").title() + "VM"
```

Comment:
This is risky, as it will replace all occurrences of "vm", not only the final one. While in the current set of classes it isn't a problem, the system is extensible and new classes can be added. For the same reason, I think forcing "VM" at the end regardless of what user provided is a bad idea.